tests/Integration/Bootstrap/Database/AbstractTester.php

```diff
@@ -0,0 +1,77 @@
+<?php
```

Comment:
So there's a package `phpunit/dbunit` that was used and is no longer maintained by the owner. Since this package is used to seed data for testing, the new files are an extract of this package providing compatibility with our codebase.

Reply:
I think it would be good open-source practice to add a header explaining this circumstance to each of these files, making sure that we credit the original project (and, by extension, their contributors).

What do you think @webong?

Reply:
I'll make this a requested change 🙏 Let me know what you think.